WebCalculator/Scripts/_references.js/CalculatorLibary.js

```diff
@@ -0,0 +1,140 @@
+﻿
+///<reference path="_references.js" />
+(function () {
+    this.calculatorNamespace = this.calculatorNamespace || {};
+    var ns = this.calculatorNamespace;
+
+    var txtInput;
+    var txtResult;
+
+    ns.initialize = function () {
+        var calculator = new ns.Calculator();
+        $('button[class^="btnNumber"]').on('click', calculator.numberClick);
+        $('#btnPlus').on('click', calculator.plusClick);
+        $('#btnMinus').on('click', calculator.minusClick);
+        $('#btnClearEntry').on('click', calculator.clearEntry);
+        $('#btnClear').on('click', calculator.clear);
+        calculator.clear();
+
+        /*for (var i = 0; i < 10; i++) {
+            document.getElementById('btn' + i).addEventListener('click', numberClick, false);
+        }
+        txtInput = document.getElementById('txtInput');
+        txtResult = document.getElementById('txtResult');
+
+        document.getElementById('btnPlus').addEventListener('click', plusClick, false);
+        document.getElementById('btnMinus').addEventListener('click', minusClick, false);
+        document.getElementById('btnClearEntry').addEventListener('click', clearEntry, false);
+        document.getElementById('btnClear').addEventListener('click', clear, false);
+        ///document.getElementById('btnequal').addEventListener('click', equal, false);
+        clear();*/
+    }
+
+    ns.Calculator = (function () {
+
+        function Calculator() {
+        }
+
+        //new function for click the number buttons
+        Calculator.prototype.numberClick = function () {
+            /*txtInput.value = txtInput.value == '0' ?
+                this.innerText : txtInput.value + this.innerText;*/
+            $('#txtInput').val($('#txtInput').val() == '0' ?
+                $(this).text() : $('#txtInput').val() + $(this).text());
```

Comment:
Make this more explicit with an `if` statement. Ternary (` condition ? value1 : value2`) statements are often confusing.